src/GraphQL/IConfigureAutoSchema.cs

```diff
@@ -12,15 +13,23 @@ public interface IConfigureAutoSchema
         /// Returns a <see cref="IGraphQLBuilder"/> reference that can be used to configure the schema or service provider.
         /// </summary>
         IGraphQLBuilder Builder { get; }
+
+        /// <summary>
+        /// Returns the type of constructed schema, which can be used to type match prior to additional configurations.
```

Comment:
```suggestion
        /// Returns the type of constructed schema, which can be used to type match prior to additional configurations.
        /// Usually it is always <see cref="AutoSchema{TQueryClrType}"/>
```

I'm not sure about what to use as a generic type inside cref tag.